src/main/java/com/cloudant/client/internal/views/ViewQueryParameters.java

```diff
@@ -277,68 +277,11 @@ protected URIBuilder getViewURIBuilder() {
                 "/_view/" + viewName);
     }
 
-    public JsonElement asJson() {
+    JsonElement asJson() {
         Map<String, Object> parameters = processParameters(gson);
         return gson.toJsonTree(parameters);
     }
 
-    public ViewQueryParameters<K, V> forwardPaginationQueryParameters(K startkey, String
```

Comment:
Just to check my understanding, this code is in PaginationToken now? Or is it just not required?